_projects/git_task_list.md

```diff
@@ -0,0 +1,87 @@
+desc: "List pull request/merge request (in gitlab) that have no reviewer yet. Provide an easy way for new comer / developer to do contribution to a project by giving review to other people pull requests."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
+  - https://github.com/coala/coala/issues/3615
+  - https://github.com/coala/coala/issues/4080
+markdown: git_task_list.md
+mentors:
+  - jayvdb
+  - yukiisbored
+  - yashovardhanagrawal
+  - sims1253
+name: "Git Task List"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "Familiarity with Git, GitHub API Gerrit API, Phabricator API, and GitLab API"
+  - "Familiarity with EmberJS, Javascript, and GitHub Pages"
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The basic idea is an extended version of [GSOC Task List][2] which was initialy
+intended for GSOC Applicans who don't have to a little experience in
+open-source. Hence, [that][1] project will be the starting point of this
+project.
+
+This project will help sharing the tasks of reviewing a pull request on a
+project. This is also maybe the solution of https://github.com/coala/coala/issues/4080,
+where developer spend too much time finding PR that haven’t been reviewed. This
+projects aims to getting more developer doing reviews works and sharing reviews
+workload, by providing an easy way to find issues that has not been reviewed.
+It also help project’s maintainer / developer do review more efficient.
+
+This also help engage developer to keep contributing to a project where they
+can’t find any issue that they can fix or keep the new comer issue for new comer
+as learning path. And developer with more experience can contribute by reviewing
+new comer’s work PR
+
+This project is also related to “New Comer Metrics And Gamification”. Where one
+of the possible accomplishment is doing reviews.
+The Goal of this projects is a serverless web page where the user can find Pull
+Request that haven’t been reviewed. User can select projects in Open Source
+hosting services like GitHub or GitLab. They also have the ability to select
+prefered programming language, tags, owner of the pull requests, and other
+metadata that is provided by the web services (Github/Gitlab).
+
+#### Milestone
+
+##### Phase 1
+- Add gitlab support
+  - Search Projects
+  - Search open issues
+  - Search issue by tag
+- Add complete list of GSOC organizations that is using github or gitlab
+  - Find gitlab / github repository pages
+  - Add simplified ui to list/search projects
+
+##### Phase 2
+- PR review list / queue
+- Create a new search page for PR
+- Search unreviewed PR
+- Search unreviewed PR on a specific projects
+
+##### Phase 3
+- Add gerit support
+- Add phabricator support
+- Improve UI and UX.
+
+#### Links
+
+- [http://github.com/summerofcode/gsoc-prep-tasks][1]
+- [http://summerofcode.github.io/gsoc-prep-tasks][2]
```

Comment:
do we need external links here? wouldn't it be better if we get a coala repo for the same...

Reply:
@yashovardhanagrawal No need, just an easy navigation. It is just my preference. For me it is easy to keep track of external stuff this way as I do skip links if Iread a long text, some kind of read later shortcut.

Reply:
See when this PR is merged, it would be up at projects.coala.io and there having a reference link to repos that are not part of coala would not be a nice idea
I think @jayvdb may be able to help us out here better

Reply:
We do need to mention the project repo URL at the top of the rendered page, not useful here.